components/feature/syncedtabs/src/test/java/mozilla/components/feature/syncedtabs/storage/SyncedTabsStorageTest.kt

```diff
@@ -95,7 +101,7 @@ class SyncedTabsStorageTest {
         // Run the flow.
         store.dispatch(TabListAction.RemoveAllPrivateTabsAction).joinBlocking()
 
-        verify(tabsStorage).store(
+        verify(tabsStorage, atLeastOnce()).store(
```

Comment:
Same comment as above.

```suggestion
        verify(tabsStorage, times(2)).store(
```